<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoCapitulo</class>
 <widget class="QWidget" name="NuevoCapitulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btCancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombreGeneral">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLugar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblLugar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Lugar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAnadirLugar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboTipoCapitulo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaestroGeneral">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maestro general</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtAsistentes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>551</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asistentes</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtInicial">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtFinal">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha final</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTomo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Tomo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>58</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>PÃ¡gs.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPaginas">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtNotas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
